book/src/csi-windows.md

```diff
@@ -0,0 +1,100 @@
+# CSI Windows Suppoort
+
+## Status
+
+Status | Min K8s Version | Min CSI proxy Version | Min Node Driver Registrar Version | latest artifacts
```

Comment:
When I preview this page, the table formatting does not seem correct

Reply:
updated